projects/Objects/HomogenousTree/src/com/valentin/HTree.java

```diff
@@ -0,0 +1,144 @@
+package com.valentin;
+
+
+import java.util.Vector;
+
+public class HTree {
+    Vector<Node> nodesVector=new Vector();
```

Comment:
access modifiers are missing and Using of vector is not required as it's synchronized and slows things down. 

> Vector is synchronized. If a thread-safe implementation is not needed, it is recommended to use ArrayList in place of Vector.